go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/cvbarros/go-teamcity
 
+go 1.14
```

Comment:
The Docker image this builds in is currently based at `golang:1.12-alpine`. Will that also need an upgrade?

Reply:
I dont think so I think my IDE auto added this.